lib/Conversation/CommentForm/CommentFormActions.js

```diff
@@ -30,7 +30,7 @@ const CommentFormActions = props =>
       </Button>
       <Button
         types={['secondary', 'size-sm']}
-        clickHandler={props.onSubmit}
+        clickHandler={() => {}}
         isSubmit
       >
         Send
```

Comment:
The form has a `onSubmit` property which calls the same function. Calling that function here causes the function to be called twice. We simply want a button that submits the form.